routes/home.js

```diff
@@ -2,18 +2,18 @@
 
 const express = require("express");
 
+const DBUtils = require("../db/utils");
+
 const router = express.Router();
 
-function handleIndexRoute(req, res) {
+async function handleIndexRoute(req, res) {
+  let breach = null;
+  if (req.query.breach) {
+    breach = await DBUtils.getBreachByName(req.query.breach);
+  }
```

Comment:
The template assumes that a breach object exists, so if we don't set a value for breach here, how about responding with an error instead of render'ing anything?

Reply:
Since this is the `home` route, we shoudn't error when the optional `breach` parameter fails. But I wrapped the template in a `{{#if breach }}` check instead.